.github/workflows/bioconda-utils-build-env-cos7.yml

```diff
@@ -0,0 +1,212 @@
+  pull_request:
+    paths:
+    - images/bioconda-utils-build-env-cos7/**
+    - .github/workflows/bioconda-utils-build-env-cos7.yml
+
+jobs:
+  build:
+    name: Build & Push
+    runs-on: ubuntu-22.04
+    env:
+      # The base image is not intended to change often and should be used with
+      # version tags or checksum IDs, but not via "latest".
+      MAJOR_VERSION: 3
+      MINOR_VERSION: 1
+      IMAGE_NAME: bioconda-utils-build-env-cos7
+
+    steps:
+    - name: Checkout bioconda-containers
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Checkout bioconda-utils
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        repository: 'bioconda/bioconda-utils'
+        path: 'images/bioconda-utils-build-env-cos7/bioconda-utils'
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+      with:
+        platforms: arm64
+
+    - name: Build
+      id: build
+      run: |
+        set -xeu
+        cd 'images/${{ env.IMAGE_NAME }}'
+
+        image_name='${{ env.IMAGE_NAME }}'
+        tags='
+          ${{ env.MAJOR_VERSION }}
+          ${{ env.MAJOR_VERSION }}.${{ env.MINOR_VERSION }}
+          latest
+        '
+        # Adds image and tags to outputs which can be used in later steps.
+        printf %s\\n \
+          "image=${image_name}" \
+          "tags=$( echo ${tags} )" \
+          >> $GITHUB_OUTPUT
+
+        # Create manifest (which is considered arch-independent)
+        for tag in ${tags} ; do
+          buildah manifest create "${image_name}:${tag}"
+        done
+
+        # Due to different nomenclature used by conda-forge and buildah, we
+        # need to map archs to base images.
+        for arch_and_image in \
+          "amd64=quay.io/condaforge/linux-anvil-cos7-x86_64" \
+          "arm64=quay.io/condaforge/linux-anvil-aarch64";
+        do
+          # Unpack archs and base images
+          arch=$(echo $arch_and_image | cut -f1 -d "=")
+          base_image=$(echo $arch_and_image | cut -f2 -d "=")
+
+          # --iidfile prints the built image ID to the specified file. This is
+          # used so we can refer to the image in later steps.
+          iidfile="$( mktemp )"
+          buildah bud \
+            --arch="${arch}" \
+            --iidfile="${iidfile}" \
+            --build-arg=base_image="$base_image"
+          image_id="$( cat "${iidfile}" )"
+          rm "${iidfile}"
+
+          # Extract various package info and version info to store as labels
+          container="$( buildah from "${image_id}" )"
+          run() { buildah run "${container}" "${@}" ; }
+          deb_list="$( run cat /.deb.lst | tr '\n' '|' | sed 's/|$//' )"
+          pkg_list="$( run cat /.pkg.lst | tr '\n' '|' | sed 's/|$//' )"
+          glibc="$( run sh -c 'exec "$( find -xdev -name libc.so.6 -print -quit )"' | sed '1!d' )"
+          bash="$( run bash --version | sed '1!d' )"
+          bioconda_utils="$( run bioconda-utils --version )"
```

Comment:
I think these two lines are needed before trying to use `bioconda-utils` - https://github.com/bioconda/bioconda-containers/pull/71/files#diff-c5b56384ebbc882b2791026dcd1a21cff80288e113a323129c82111c4351cef8R3-R4